### PR TITLE
Web3 World Hackaton - Add RandomList.vue

### PR DESCRIPTION
**Pull request by antodp**
## Description
My work consists in the creation of a component named `RandomList.vue`.
The component shows a custom component (default component is an unordered list) passing it via a prop the randomly selected data. The random selection is accomplished shuffling the list and slicing it.
An example of usage is visible in component `RandomAppList.vue`, displayed in the endpoint `/use/`.

### CODE REVIEW

**Comment by samajammin** [comment]
Thanks for your submission! Will review ASAP - definitely by end of this week.

File: docs/.vuepress/components/RandomAppList.vue [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<template>
+	<RandomList :n="4", :items="items"></RandomList>
```
Comment (samajammin) [inline docs/.vuepress/components/RandomAppList.vue]: Please remove the comma between props here. It's causing this console error when navigating to the /use page:
```
DOMException: Failed to execute 'setAttribute' on 'Element': ',' is not a valid attribute name.
```

Comment (antodp) [reply docs/.vuepress/components/RandomAppList.vue]: Okay, done!

**Comment by samajammin** [comment]
Thanks! 🎉

**Pull request merged**